_data/Tools/src/runner.py

```diff
@@ -29,13 +30,122 @@ def run(self):
                 if vorID in newData.keys():  # only rewrite if the VOR/DME is in both the old data and the new data, otherwise existing data is kept
                     # if the VOR/DME is in both the old data and the new data, write the new data onto the old data (if the data is the same we still write, just no change will be visible because the written data is the same as the stored data)
                     dataAboutVORDME = newData[vorID]
-                    currentData[i] = f"{vorID} {dataAboutVORDME[1]} {' '.join(dataAboutVORDME[2])} ; {dataAboutVORDME[0]}"
+                    currentData[i] = f"{vorID} {dataAboutVORDME['frequency']} {' '.join(dataAboutVORDME['coordinates'])} ; {dataAboutVORDME['name']}"
 
             self.writeLines("Navaids/VOR_UK.txt", currentData)
+        
+        elif self.args["page"] == "ENR4.4" or self.args["page"] == "all":
+            currentData = self.readCurrentData("Navaids/FIXES_UK.txt")
+
+            newData = self.aipApi.parseENR4_4()
+
+            for i, line in enumerate(currentData):
+                fixID = line.split(" ")[0]
+                if fixID in newData.keys():
+                    dataAboutFix = newData[fixID]
+                    currentData[i] = f"{fixID} {' '.join(dataAboutFix['coordinates'])}"
+
+            self.writeLines("Navaids/FIXES_UK.txt", currentData)
+        
+        elif self.args["page"] == "ENR3.2" or self.args["page"] == "all":
+            newData = self.aipApi.parseENR3_2()
+
+            lowerAirways = os.listdir("../../Airways/RNAV/Lower")
+            upperAirways = os.listdir("../../Airways/RNAV/Upper")
```

Comment:
Do these need changing to `ATS Routes` since the folder name was changed?

Reply:
This is why I should merge the branch before pushing changes 🤦